.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -15,24 +15,34 @@ on:
 concurrency:
   # A PR number if a pull request and otherwise the commit hash. This cancels
   # queued and in-progress runs for the same PR (presubmit) or commit
-  # (postsubmit). The workflow name is prepended to avoid conflicts between
-  # different workflows.
+  # (postsubmit)
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: arc-runner-set
```

Comment:
Can we call this `azure-windows-scale`, to match the existing `azure-linux-scale`? The name `arc-runner-set` doesn't suggest anything about the runner platform, size, features, etc.

How much effort would it be to rename it?